app/code/Magento/Quote/Test/Unit/Model/Quote/Address/Total/SubtotalTest.php

```diff
@@ -229,13 +239,14 @@ public function testCollectWithInvalidItems()
         $address->expects($this->once())
             ->method('removeItem')
             ->with($addressItemId);
-        $addressItem = $this->createPartialMock(
-            AddressItem::class,
-            [
-                'getId',
-                'getQuoteItemId'
-            ]
-        );
+        $addressItem = $this->getMockBuilder(AddressItem::class)
+            ->disableOriginalConstructor()
+            ->disableOriginalClone()
+            ->disableArgumentCloning()
+            ->disallowMockingUnknownTypes()
```

Comment:
**YAGNI** If original code is not performing any `clone` then you don't need to provide these methods.